### PR TITLE
Improve English spelling

### PR DESCRIPTION
**Pull request by hazzik**
~I'm not sure yet about the spelling variations proposed here, so please review and make suggestions.~ I've reviewed some doubtful places with native speaker, so it's ready for review.

### CODE REVIEW

File: locales/English.lng [hunk 1]
```diff
@@ -468,13 +468,13 @@
 467=Research
 468=Task
 469=Correspondence
-470=Correction IDs
-471=Check format
+470=Correcting IDs
```
Comment (hazzik) [inline locales/English.lng:471]: 470 and 471 are progress labels so continuous tense is more appropriate

File: locales/English.lng [hunk 2]
```diff
@@ -705,7 +705,7 @@
 704=Type of comparison
 705=Internal comparing (search for duplicates)
 706=Comparing with another database
-707=Analysis
+707=Analyze
```
Comment (hazzik) [inline locales/English.lng:708]: This is an button, so imperative would be more appropriate.

File: projects/GKv2/GKPedigreeImporterPlugin/GKPedigreeImporterPlugin.enu [hunk 3]
```diff
@@ -27,7 +27,7 @@ Next >
 Close
 Special settings of personal lines format
 1) "Number. Name (*birth +death)" - the dates in brackets
-Analysis
+Analyzing
```
Comment (hazzik) [inline projects/GKv2/GKPedigreeImporterPlugin/GKPedigreeImporterPlugin.enu:30]: This is on other hand is a progress label, so continuous tense would be more appropriate.

File: projects/GKCore/GKCore/LangMan.cs [hunk 4]
```diff
@@ -1673,7 +1673,7 @@ public static class LangMan
             /* 781 */ "Adopted",
             /* 782 */ "Birth",
             /* 783 */ "Foster",
-            /* 784 */ "Charset detection at load",
+            /* 784 */ "Detect Charset at load",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1676]: This is checkbox in options. Imperative mood sounds more appropriate here.

File: projects/GKCore/GKCore/LangMan.cs [hunk 5]
```diff
@@ -1637,7 +1637,7 @@ public static class LangMan
             /* 745 */ "Show default portraits",
             /* 746 */ "A media file is added from the removable media. Continue?",
             /* 747 */ "Warn when adding media files from removable media",
-            /* 748 */ "Load at startup recent opened files",
+            /* 748 */ "Load recent opened files at startup",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1640]: More natural word order

File: projects/GKCore/GKCore/LangMan.cs [hunk 6]
```diff
@@ -1359,13 +1359,13 @@ public static class LangMan
...
-            /* 470 */ "Correction IDs",
-            /* 471 */ "Check format",
+            /* 470 */ "Correcting IDs",
+            /* 471 */ "Checking format",
             /* 472 */ "Correction of identity records is required, continue?", // obsolete
             /* 473 */ "Number of objects in the master database: {0}",
             /* 474 */ "Synchronization is complete.",
-            /* 475 */ "Search the patriarchs",
-            /* 476 */ "Search links",
+            /* 475 */ "Searching the patriarchs",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1367]: 475 and 476 Progress labels

File: projects/GKCore/GKCore/LangMan.cs [hunk 7]
```diff
@@ -1408,7 +1408,7 @@ public static class LangMan
             /* 516 */ "Search unlinked namesakes",
             /* 517 */ "Check connection of families",
             /* 518 */ "Check database",
-            /* 519 */ "Search the patriarchs",
+            /* 519 */ "Find the patriarchs",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1411]: A menu item. Probably original is more appropriate. I need to research about "find" vs "search" semantic.

File: projects/GKCore/GKCore/LangMan.cs [hunk 8]
```diff
@@ -1276,7 +1276,7 @@ public static class LangMan
             /* 384 */ "<?>",
             /* 385 */ "Next",
             /* 386 */ "You know that ...",
-            /* 387 */ "Load and search for places",
+            /* 387 */ "Loading and searching places",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1279]: Probably `Loading and searching the places`.

File: projects/GKCore/GKCore/LangMan.cs [hunk 9]
```diff
@@ -991,7 +991,7 @@ public static class LangMan
             /* 099 */ "Cancel",
             /* 100 */ "Close",
             /* 101 */ "Select",
-            /* 102 */ "Append",
+            /* 102 */ "Add new",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:994]: This is a button in record selection dialog. "Append" does not correctly describe the action that will be performed by the button.

File: projects/GKCore/GKCore/LangMan.cs [hunk 10]
```diff
@@ -1596,7 +1596,7 @@ public static class LangMan
             /* 704 */ "Type of comparison",
             /* 705 */ "Internal comparing (search for duplicates)",
             /* 706 */ "Comparing with another database",
-            /* 707 */ "Analysis",
+            /* 707 */ "Analyze",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1599]: This is a button, so imperative would be more appropriate.

File: projects/GKCore/GKCore/LangMan.cs [hunk 11]
```diff
@@ -1359,13 +1359,13 @@ public static class LangMan
             /* 467 */ "Research",
             /* 468 */ "Task",
             /* 469 */ "Correspondence",
-            /* 470 */ "Correction IDs",
-            /* 471 */ "Check format",
+            /* 470 */ "Correcting IDs",
```
Comment (hazzik) [inline projects/GKCore/GKCore/LangMan.cs:1362]: 470 and 471 are progress labels so continuous tense is more appropriate

File: projects/GKCore/GKCore/LangMan.cs (hunk 9 above)
Comment (Serg-Norseman) [reply projects/GKCore/GKCore/LangMan.cs:994]: 👍

File: projects/GKCore/GKCore/LangMan.cs (hunk 10 above)
Comment (Serg-Norseman) [reply projects/GKCore/GKCore/LangMan.cs:1599]: 👍

File: projects/GKCore/GKCore/LangMan.cs (hunk 4 above)
Comment (Serg-Norseman) [reply projects/GKCore/GKCore/LangMan.cs:1676]: 👍

**Review by Serg-Norseman — APPROVED** [review]
Good! 👍

**Pull request merged**